tools/generate-pdfs.sh

```diff
@@ -16,13 +16,22 @@ set -ex
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-mkdir -p pdfs
+function generate_pdfs_from_md() {
+	inputMdFile=$1
+	if ! [ -f "$inputMdFile" ]; then
+		echo "**** WARNING! $inputMdFile does not exist. Please enter valid file path."
```

Comment:
Can we add an extra warning line that informs the users what to do in case of failure? Something along the lines of `please make sure to run `tools/generate-intrinsics-specs.sh` before building the PDFs.

Reply:
Just an echo command following the one on line 22